### PR TITLE
Mission biblio ref

### PR DESCRIPTION
**Pull request by koalie**
I'm not sure if this reference is right for "DNS", or if it's just missing its link; flagging as a bug.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -740,7 +740,7 @@ <h5 class="heading settled" data-level="2.3.3.1" id="architecture"><span class="
...
+   <p>Another pillar of important element of this architecture are the <strong>Decentralized Identifiers (DIDs)</strong>, a new Uniform Resource Identifier (URI) type that allows entities and resources to be identified through various methods <a data-link-type="biblio" href="#biblio-did-core" title="Decentralized Identifiers (DIDs) v1.0">[did-core]</a>. These methods can rely on various technologies, including blockchains such as Bitcoin or Ethereum, the web, InterPlanetary File System (IPFS), and Domain Name System (DNS) @@missing reference: [did-spec-registries]. All methods allow cryptographical control over the identifier. Still, they vary in how decentralized they are and the features they support (e.g., key rotation and revocation).</p>
```
Comment (simoneonofri) [inline index.html:743]: it is the refernce for "These methods", often i put the reference at the end of the sentence. May I need to move upfront?

Comment (koalie) [reply index.html:743]: Ah, I see! 
I don't know!
In any case, there is no link on the reference, wherever you put it.

Comment (simoneonofri) [reply index.html:743]: oh, i used the specref for that, i need to check the `.bs` file and fix

Comment (simoneonofri) [reply index.html:743]: checked, added `[[`

**Pull request closed without merging**